randovania/games/samus_returns/logic_database/Area 6.txt

```diff
@@ -33,6 +33,14 @@ Extra - asset_id: collision_camera_034
       All of the following:
           Morph Ball
           Spider Ball or Defeat Non-Counterable Enemies
+  > Lower Tunnels
```

Comment:
This node on the map is weird.  If its just right before `before diggernaut arena` then why is there even a connection from save -> `before diggernaut arena`? Leads me to also being confused on the requirements here

Reply:
I dislike that entire section if I'm honest. I wanted to simplify it since there are too many nodes imo

Reply:
Looks like I forgot to remove the connection.